### PR TITLE
FF-2283 determine enable client-side logging when "widnow" is available

### PR DESCRIPTION
**Pull request by greghuels**
---
labels: mergeable
---
[//]: #  (Link to the issue corresponding to this chunk of work)
Fixes: #__issue__

## Motivation and Context
[//]: #  (Why is this change required? What problem does it solve?)

## Description
[//]: # (Describe your changes in detail)

## How has this been tested?
[//]: # (Please describe in detail how you tested your changes)

[//]: # (OPTIONAL)
[//]: # (Add one or multiple labels: enhancement, refactoring, bugfix)

### CODE REVIEW

File: src/application-logger.ts [hunk 1]
```diff
@@ -2,9 +2,11 @@ import pino from 'pino';
...
 export const loggerPrefix = '[Eppo SDK]';
 
+const isNode = typeof window === 'undefined';
+
 // Create a Pino logger instance
 export const logger = pino({
   level: process.env.NODE_ENV === 'production' ? 'warn' : 'info',
   // https://getpino.io/#/docs/browser
-  browser: { disabled: true },
+  browser: { disabled: isNode },
```
Comment (greghuels) [inline src/application-logger.ts:11]: @leoromanovsky @aarsilv It looks like Pino does support client-side logging, so it seems like we just need to detect to see if we're running in the browser or not.

Comment (greghuels) [reply src/application-logger.ts:11]: not sure yet -- let me do a bit of testing to verify

Comment (greghuels) [reply src/application-logger.ts:11]: > however at compile-time is where we are seeing the error
Did the client say this? It looks like the error contains `webpack-internal://` in the trace, which could also be a runtime error coming from https://webpack.js.org/configuration/dev-server/

Comment (leoromanovsky) [reply src/application-logger.ts:11]: I saw you on the customer thread thanks for asking questions - I might have misunderstood where their error was coming from.

Comment (aarsilv) [reply src/application-logger.ts:11]: Oh nice spot!

Will still be good to reproduce and then verify this fixes it

**Pull request closed without merging**